charts/amazon-cloudwatch-observability/templates/linux/cloudwatch-agent-custom-resource.yaml

```
@@ -233,7 +233,7 @@ spec:
     value: "True"
   {{ end }}
   - name: K8S_CLUSTER_NAME
-    value: {{ $.Values.clusterName }}
+    value: "{{ $.Values.clusterName }}"
```

To keep it consistent with our other code.
```suggestion
    value: {{ $.Values.clusterName | quote }}
```

---

Looks good. I saw it done both ways in the helm chart. But either way is just as good to me